harness/features/initialize.cloud.test.ts

```diff
@@ -25,7 +25,10 @@ describe('Client Initialize Tests - Cloud', () => {
                 const client = new CloudTestClient(name)
                 const response = await client.createClient({}, 'invalidKey', true)
                 const body = await response.json()
-                expect(body.exception).toBe('Invalid environment key provided. Please call initialize with a valid server environment key')
+                expectErrorMessageToBeTheSame(
```

Comment:
maybe just `expectErrorMessageToBe`?

Reply:
with that wording it makes me think there's only going to be one param tho

Reply:
I dont read it that way, to me its just the same as when you write out the `expect` statement